src/index.js

```diff
@@ -11,8 +11,12 @@ const createGroups = (pages, pageLength) =>
 
 const getPageIndex = index => (index === 0 ? `` : index + 1)
 
-const buildPaginationRoute = (index, pathPrefix) =>
-  index > 1 ? `${pathPrefix}/${index}` : pathPrefix
+const buildPaginationRoute = (index, pathPrefix) => {
+  if (!index && !path) {
+    return `/`
+  }
+  return index > 1 ? `${pathPrefix}/${index}` : pathPrefix
```

Comment:
How about we just do
```js
return index > 1 ? `${pathPrefix}/${index}` : (pathPrefix || `/`)
```
and leave out the
```js
if (!index && !path) {
   ...
}
```
check completely?

Reply:
sounds good!